auto_install/install.sh

```diff
@@ -761,9 +760,12 @@ setCustomDomain() {
 }
 
 confOpenVPN() {
-    # Generate a random, alphanumeric identifier of 16 characters for this server so that we can use verify-x509-name later that is unique for this server installation. Source: Earthgecko (https://gist.github.com/earthgecko/3089509)
-    NEW_UUID=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 16 | head -n 1)
-    SERVER_NAME="server_${NEW_UUID}"
+    # Grab the existing Hostname
+	HOST_NAME=$(hostname -s)
+	# Generate a random UUID for this server so that we can use verify-x509-name later that is unique for this server installation.
+    $NEW_UUID=$(</proc/sys/kernel/random/uuid)
```

Comment:
Little syntax, no `$` for variable declaration:
```
NEW_UUID=$(</proc/sys/kernel/random/uuid)
```

And not important, but the space/alignment show different on GitHub. You used `<tab>` character, which I bet has 4 spaces on most editors, but on GitHub those are shown as 8 spaces. In the surrounding code, instead of `<tab>`, there are 4 single spaces used instead.